src/lib/user.svelte.ts

```diff
@@ -103,8 +106,17 @@ let catalogId: {
   };
 });
 
+let isLoginDialogOpen = $state(false);
+
 /** The user store. */
 export const user = {
+  get isLoginDialogOpen() {
+    return isLoginDialogOpen;
+  },
+  set isLoginDialogOpen(value) {
+    isLoginDialogOpen = value;
+  },
+
```

Comment:
@michaelwschultz I rebased and pushed an extra commit to move the `isLoginDialogOpen` into the user store and just wanted to make sure that seemed good to you before merging.